src/renderer/components/dex/MarketSelection.vue

```diff
@@ -62,13 +62,40 @@ export default {
           return result;
         }, []);
     },
+
+    storeStateCurrentMarket() {
+      return this.$store.state.currentMarket;
+    },
+  },
+
+  data() {
+    return {
+      currentMarket: {},
+    };
   },
 
+
   methods: {
     toggleNightMode() {
       this.$store.commit('setStyleMode', this.$store.state.styleMode === 'Night' ? 'Day' : 'Night');
     },
   },
+
+  mounted() {
+    this.currentMarket = this.$store.state.currentMarket;
+  },
+
+  watch: {
+    currentMarket(newVal) {
+      this.$store.commit('setCurrentMarket', newVal);
+    },
+
+    storeStateCurrentMarket(newVal) {
+      if (!this.currentMarket) {
```

Comment:
Why does this only set the currentMarket if it is falsy?

Reply:
Not sure, this was something @jameswhollister wrote I believe.